src/instructlab/model/download.py

```diff
@@ -191,6 +191,15 @@ def download(self):
             f"Downloading model from OCI registry: {self.repository}@{self.release} to {self.download_dest}..."
         )
 
+        # raise an exception if user specified tag/SHA embedded in repository instead of specifying --release
```

Comment:
(Just a thought) if you already successfully extracted the necessary tag from the string, why not just use it? (maybe warn but proceed?)

Reply:
I'm not opposed to that
I was just wondering if it creates a more confusing UX by introducing inconsistencies between how OCI and HF downloads work

Reply:
(Evil thought) Maybe HF should also extract out the tag! :D

Reply:
sure - maybe as a post GA thing? the UX could use a lot of cleaning up and improvement in general

Reply:
Oh I didn't mean this to block the PR! Just a curiosity. Feel free to ignore.

Reply:
Yeah IMO we should revisit in the future